radix-engine/src/system/checkers/kernel_db_checker.rs

```diff
@@ -14,7 +14,6 @@ pub enum KernelDatabaseCheckError {
     NoOwnerForNonGlobalNode(NodeId),
     ZeroPartitionCount(NodeId),
     CannotOwnBootLoaderNode(NodeId),
```

Comment:
Should remove `CannotOwnBootLoaderNode` as well?

Reply:
Yup good catch